elements/pfe-tabs/src/pfe-tab-panel.scss

```diff
@@ -46,29 +64,32 @@ $LOCAL-VARIABLES: (
 /// HORIZONTAL
 /// ===========================================================================
 :host([pfe-variant="earth"]) {
-  --pfe-tabs__panel--Padding: calc(#{pfe-var(container-spacer)} * 2);
-
-  --pfe-tabs__panel--BorderRight: #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} #{pfe-var(surface--border)};
-  --pfe-tabs__panel--BorderBottom: #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} #{pfe-var(surface--border)};
-  --pfe-tabs__panel--BorderLeft: #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} #{pfe-var(surface--border)};
-
 
   @include browser-query(ie11) {
     .container {
+      background-color: pfe-fetch(surface--lightest);
       padding: 1em;
-      border-right: 1px solid #{pfe-var(surface--border)};
-      border-bottom: 1px solid #{pfe-var(surface--border)};
-      border-left: 1px solid #{pfe-var(surface--border)};
+      border-right: 1px solid pfe-fetch(surface--border);
+      border-bottom: 1px solid pfe-fetch(surface--border);
+      border-left: 1px solid pfe-fetch(surface--border);
     }
   }
+
+}
+:host([vertical][pfe-variant="earth"]) {
```

Comment:
Vertical is listed below on line 84, should this live there?

Reply:
Merge this with line 101 perhaps?

Reply:
moved